cmake/toolchain/xtools.cmake

```diff
@@ -6,8 +6,13 @@ set(TOOLCHAIN_HOME ${XTOOLS_TOOLCHAIN_PATH})
 
 set(COMPILER gcc)
 
-set(CROSS_COMPILE_TARGET_arm arm-none-eabi)
-set(CROSS_COMPILE_TARGET_x86 i586-zephyr-elfiamcu)
+set(CROSS_COMPILE_TARGET_arm arm-zephyr-eabi)
+set(CROSS_COMPILE_TARGET_x86 i586-zephyr-elf)
```

Comment:
note that we still need to solve using -elf and -elfiamcu on various x86 targets, but at least defaulting to -elf makes my life much easier.